modin/core/io/column_stores/hdf_dispatcher.py

```diff
@@ -69,11 +68,12 @@ def _read(cls, path_or_buf, **kwargs):
             Query compiler with imported data for further processing.
         """
         if cls._validate_hdf_format(path_or_buf=path_or_buf) is None:
-            ErrorMessage.default_to_pandas(
+            return cls.single_worker_read(
+                path_or_buf,
                 "File format seems to be `fixed`. For better distribution consider "
-                + "saving the file in `table` format. df.to_hdf(format=`table`)."
+                + "saving the file in `table` format. df.to_hdf(format=`table`).",
+                **kwargs
             )
```

Comment:
I would rather you pass the message as named argument (here and everywhere else), i.e.
```pythonё
return cls.single_worker_read(path_or_buf, reason="File format ......", **kwargs)
```

I think this improved readability